src/libespeak-ng/fifo.c

```diff
@@ -131,6 +131,9 @@ espeak_ng_STATUS fifo_add_command(t_espeak_command *the_command)
 espeak_ng_STATUS fifo_add_commands(t_espeak_command *command1, t_espeak_command *command2)
 {
 	espeak_ng_STATUS status;
+	if (!my_thread) {
+		return ENS_NOT_INITIALIZED;
+	}
```

Comment:
This will break on systems where pthread_t can be 0 for some threads.

Reply:
I don't know any platform where pthread_t may be zero. But OK, I will rewrite this on another `bool` variable.